### ISSUE TITLE
您好，我将tf.nn.sparse_softmax_cross_entropy_with_logits改为focal-loss时出现了以下问题：

### ISSUE THREAD

**Issue by lilmangolil** [fudannlp16/focal-loss#5]
InvalidArgumentError (see above for traceback): tags and values not the same shape: [] != [256] ，请问怎样解决？谢谢(*^▽^*)

**Comment by jamiechoi1995** [comment]
Add tf.reduce_mean.
example:
before:
loss = focal_loss_softmax(flaten_cls_targets, cls_pred) * (params['negative_ratio'] + 1.)
after:
loss = tf.reduce_mean(focal_loss_softmax(flaten_cls_targets, cls_pred) * (params['negative_ratio'] + 1.))
